src/ctlib/ct.c

```diff
@@ -3372,7 +3372,10 @@ ct_param(CS_COMMAND * cmd, CS_DATAFMT * datafmt, CS_VOID * data, CS_INT datalen,
 		}
 
 		param = tds_new0(CSREMOTE_PROC_PARAM, 1);
-
+		if(!param){
+			printf("Could not allocate memory to param\n");
```

Comment:
libraries should not output on standard error, each library (in this case CT-Library) has a way to report errors.